iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/PublicAccessCheck.java

```diff
@@ -71,8 +80,25 @@ protected void registerResourceConsumer() {
       resource -> resource.attribute("visibility")
         .reportIf(equalTo("public"), MESSAGE)
         .reportIfAbsent(OMITTING_DNS));
-  }
 
+    register("google_container_cluster",
+      resource -> {
+        BlockSymbol config = resource.block("private_cluster_config");
+        config.reportIfAbsent(OMITTING_KUBERNETES);
```

Comment:
I thought you were not `String.format`-ing the right message here, but then I looked at the implementation of `ifAbsent(...)`

But if it is *Absent* - I think you should break here (otherwise you will raise another issue down the line) ?

Reply:
I don't get your question. Please clarify.

Reply:
Well, obviously the tests prove my concerns are in vein ...
I don't understand how is it that if `private_cluster_config` is missing then we don't also raise on line 92 (right after the `if (nodes.isAbsent() && endpoint.isAbsent())` )